src/main/steps/common/task-list/utils.ts

```diff
@@ -28,7 +28,7 @@ export const getPartyName = (
   return partyDetails ? `${partyDetails.firstName} ${partyDetails.lastName}` : '';
 };
 
-export const isCaseWithdrawn = (caseData: Partial<CaseWithId>): boolean => {
+export const isCaseWithdrawn = (caseData: Partial<CaseWithId> | undefined): boolean => {
```

Comment:
@sealDebolina Why are we modifying the code logic for test cases ?
logic should be unaltered, please revert this.

Reply:
Sure,
for next line(if (!caseData) ) checks if CaseData is undefined, for that reason I had done.
if next line is meant to check empty data I think we should use if (Object.keys(caseData).length === 0)